jquery.unveil.js

```diff
@@ -16,14 +16,20 @@
         th = threshold || 0,
         retina = window.devicePixelRatio > 1,
         attrib = retina? "data-src-retina" : "data-src",
+        bkg    = retina? "data-bkg-retina" : "data-bkg",
         images = this,
         loaded;
 
     this.one("unveil", function() {
-      var source = this.getAttribute(attrib);
-      source = source || this.getAttribute("data-src");
+      var bkg_source = this.getAttribute(bkg);
```

Comment:
Fallback if no retina background specified:
var bkg_source = this.getAttribute(bkg) || this.getAttribute("data-bkg");